### ISSUE TITLE
Can this write blank nodes with multiple predicate/object pairs?

### ISSUE THREAD

**Issue by edwardsph** [rdfjs/N3.js#360]
The output I am looking for is:
```
[ ] <a> <b> ;
    <c> <d> .
```
or
```
[
  <a> <b> ;
  <c> <d> 
] .
```
The workaround I have found is obviously not intended as it uses internal functions:
```
writer._write(writer._encodeIriOrBlank(writer.blank([
  { predicate: namedNode('a'), object: namedNode('b') },
  { predicate: namedNode('c'), object: namedNode('d') },
])));
writer._write('.\n');
```
This produces the second style output.

**Comment by jeswr** [comment]
I wouldn't think so as the design goal of this library is stream writing rather than pretty-printing.

**Comment by RubenVerborgh** [comment]
> The output I am looking for is:
> 
> ```turtle
> [ ] <a> <b> ;
>     <c> <d> .
> ```

This one should have worked:

```javascript
import { Writer, DataFactory } from './lib/index.js';

const writer = new Writer();
const { namedNode } = DataFactory;

const blank = writer.blank();
writer.addQuad(blank, namedNode('a'), namedNode('b'));
writer.addQuad(blank, namedNode('c'), namedNode('d'));

writer.end((err, result) => console.log(result));
```

but it returns this at the moment:

```turtle
[] <a> <b>.
[] <c> <d>.
```

Let me check if this is an easy fix.

**Comment by RubenVerborgh** [comment]
Fix published as v1.17.1.
